spec/requests/devise_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe 'Devise Token Auth', type: :request do
+  it 'registers a user' do
+    headers = { 'ACCEPT' => 'application/json' }
+    params = {
+      email: 'florencia.mosquera@rootstrap.com',
+      password: 'password',
+      password_confirmation: 'password',
+      gender: :female
+    }
```

Comment:
Use [`#let`](https://relishapp.com/rspec/rspec-core/v/2-11/docs/helper-methods/let-and-let)